### PR TITLE
[fontsize] Avoid creating dummy labels for determining line height. C…

### PR DESCRIPTION
**Pull request by jpetrell**
…ontributes to JB#14363

### CODE REVIEW

File: qml/CoverPage.qml [hunk 1]
```diff
@@ -63,10 +63,9 @@ CoverBackground {
             text: model.fileName
             iconSource: window.mimeToIcon(model.fileMimeType)
         }
-        Label {
-            id: dummy
-            visible: false
-            text: "Dummy"
+        FontMetrics {
+            id: fontMetrics
+            font.pixelSize: Theme.fontSizeMedium
```
Comment (rainemak) [inline qml/CoverPage.qml]: Will ```FontMetrics``` read default font family or should that be defined here as well?
```
font {
    pixelSize: Theme.fontSizeMedium
    family: Theme.fontFamily
```

Comment (rainemak) [inline qml/CoverPage.qml]: Replying to myself... of course FontMetrics must read default font family.

File: rpm/sailfish-office.spec [hunk 2]
```diff
@@ -16,7 +16,7 @@ BuildRequires: cmake
 BuildRequires: qt5-qttools-linguist
 Requires: calligra-components >= 2.7.9+git4
 Requires: calligra-filters >= 2.7.9+git4
-Requires: sailfishsilica-qt5 >= 0.21.63
+Requires: sailfishsilica-qt5 >= 0.23.14
```
Comment (rainemak) [inline rpm/sailfish-office.spec]: I don't see that this commit/change would need this requirement. Qt5.6 (quick) on the other hand is required. (I'm seeing indirect reasoning).

**Comment by pvuorela** [comment]
Concur that dependency is really on qt quick, not Silica, but indirectly it ensures new enough version already so good enough for me. LGTM.

**Comment by jpetrell** [comment]
I'll add correct dependency.

**Pull request merged**